### PR TITLE
Dfl 9432 aside blocks

### PR DESCRIPTION
**Pull request by kevinhowbrook**
# Description

This is based off https://github.com/mozilla/foundation.mozilla.org/pull/9434

This PR adds a new 'aside block' to be used on the banner page type:

The FE looks like this (Note my FE skills are not good enough to correct whats happening with the link blocks in the aside area)

Screenshot(27)

In the admin area, aside blocks are added at the bottom of the page:

image

Link to sample test page: http://localhost:8000/en/data-future-labs/
Related PRs/issues: #9432 

# Checklist

**Tests**
- [ ] ~~Is the code I'm adding covered by tests?~~

**Changes in Models:**
- [ ] ~~Did I update or add new fake data?~~
- [ ] ~~Did I squash my migration?~~
- [ ] ~~Are my changes [backward-compatible](https://github.com/mozilla/foundation.mozilla.org/blob/main/docs/workflow.md#django-migrations-what-to-do-when-working-on-backward-incompatible-migrations). If not, did I schedule a deploy with the rest of the team?~~

**Documentation:**
- [ ] ~~Is my code documented?~~
- [ ] ~~Did I update the READMEs or wagtail documentation?~~

### CODE REVIEW

**Comment by tbrlpld** [comment]
@kevinhowbrook The changes in chained/stacked PRs become easier to review if you point the PR to be merged into the branch that it is based. Github should be smart enough I think to update the PR once to the target of the previous PR once that previous PR/branch is merged.

File: network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/aside.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{% load wagtailcore_tags %}
+
+{% block block_head %}{% endblock %}
+
+<div class="tw-container {{block.block_type}}-block {% block block_container_classes %}{% endblock %} tw-border-t tw-border-black tw-py-2">
```
Comment (tbrlpld) [inline network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/aside.html:5]: **Question** Do we need the `tw-container` and the nested `tw-row`?

**Review by tbrlpld — COMMENTED** [review]
@kevinhowbrook Looking good. I have a few questions around the styling. I think we can probably simplify this template a bit. 

I am not sure what is going on with the link block either. @danielfmiranda @fessehaye do you know what is happening here? You are the last two who have touched the template. It looks ok in the style guide.

File: network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/aside.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (kevinhowbrook) [reply network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/aside.html]: These were copied over from the other block templates but they aren't being used here so I've removed them in https://github.com/mozilla/foundation.mozilla.org/commit/8476666c1f62885cb71daebcbcb06fbb489f1c17

File: network-api/networkapi/wagtailpages/templates/wagtailpages/campaign_page.html [hunk 2]
```diff
@@ -92,7 +92,7 @@ <h1 class="tw-h1-heading {% if uses_menu %}tw-hidden medium:tw-block{% endif %}"
       </div>
 
     {% if page.aside %}
-      <div class="medium:tw-w-1/4">
+      <div class="aside medium:tw-w-1/4 tw-px-4 tw-mt-4">
```
[the remark this answers is not in the document]
Comment (kevinhowbrook) [reply network-api/networkapi/wagtailpages/templates/wagtailpages/campaign_page.html]: :+1:

File: network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/aside.html [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  <div class="tw-row tw-justify-center {% block block_row_classes%}{% endblock %}">
+    <div class="{% block main_block_class %}streamfield-content{% endblock%} {% block streamfield_content_classes%}{% endblock %}">
```
[the remark this answers is not in the document]
Comment (kevinhowbrook) [reply network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/aside.html]: removed in 0a7d97582601fab2407d5acf995981a73c381bef

File: network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/aside.html (hunk 1 above)
Comment (kevinhowbrook) [reply network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/aside.html]: Thanks for questioning these, as above, I've cleaned this up in 0a7d97582601fab2407d5acf995981a73c381bef

I was of the opinion the template should match the other block templates and provide the same overrides and markup but really none of them are needed so I've removed them

**Pull request merged**